kubernetes/yamls/development/api-server/api-deployment.yml

```diff
@@ -18,13 +18,8 @@ spec:
           httpGet:
             path: /api/status.json
             port: 80
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-        readinessProbe:
-          httpGet:
-            path: /api/status.json
-            port: 80
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
+          periodSeconds: 300
           timeoutSeconds: 3
```

Comment:
* How do you know that Elasticsearch deployment is running fine? Can you please check the logs for Elasticsearch?
* Have you created the persistent disk?
* What is transport address pointing to in the configs?
    ```bash
     kubectl exec -it <pod> --namespace=web -c server bash
     # You are now in server pod
     cat loklak_server/conf/config.properties  # Look for transport address
    ```